test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/ReviewRegistrationControllerSpec.scala

```diff
@@ -213,6 +206,47 @@ class ReviewRegistrationControllerSpec extends ControllerSpec {
         ).getCount mustBe 1
       }
 
+      "user is not showed confirmation page if registration has no incorporation details for limited company" in {
```

Comment:
A small typo here, should probably be `shown` rather than `showed`?

Reply:
Ditto for other instances of this through the rest of this file.

Reply:
Fixed :)